src/SDKs/KeyVault/dataPlane/README.md

```diff
@@ -0,0 +1,78 @@
+    - Create, Read, Update, and Delete Keys
+    - Import, Backup and Restore Keys
+    - Encrypt, Decrypt, Wrap, Unwrap, Sign and Verify cryptographic Key operations 
+- Certificates
+    - Create, Read, Update, and Delete Certificates
+    - Create, Read, Update, and Delete certificate renewal properties
+    - Create, Read, Update, and Delete certificate issuers
+
+## Getting Started
+
+The complete Microsoft Azure SDK can be downloaded from the [Microsoft Azure Downloads Page](http://azure.microsoft.com/en-us/downloads/?sdk=net) and ships with support for building deployment packages, integrating with tooling, rich command line tooling, and more.
+
+Please review [Get started with Azure Key Vault](https://docs.microsoft.com/en-us/azure/key-vault/key-vault-get-started) if you are not familiar with Azure Key Vault.
+
+For the best development experience, developers should use the official Microsoft NuGet packages for libraries. NuGet packages are regularly updated with new functionality and hotfixes. 
+
+## Requirements
+
+- Microsoft Azure Subscription: To call Microsoft Azure services, you need to first [create an account](https://account.windowsazure.com/Home/Index). Sign up for a free trial or use your MSDN subscriber benefits.
+- Hosting: To host your .NET code in Microsoft Azure, you additionally need to download the full Microsoft Azure SDK for .NET - which includes packaging,
+    emulation, and deployment tools, or use Microsoft Azure Web Sites to deploy ASP.NET web applications.
+
+## Download Packages
+
+- [Microsoft.Azure.KeyVault](https://www.nuget.org/packages/Microsoft.Azure.KeyVault)
+- [Microsoft.Azure.KeyVault.Core](https://www.nuget.org/packages/Microsoft.Azure.KeyVault.Core)
+- [Microsoft.Azure.KeyVault.WebKey](https://www.nuget.org/packages/Microsoft.Azure.KeyVault.WebKey)
+- [Microsoft.Azure.KeyVault.Cryptography](https://www.nuget.org/packages/Microsoft.Azure.KeyVault.Cryptography)
+- [Microsoft.Azure.KeyVault.Extensions](https://www.nuget.org/packages/Microsoft.Azure.KeyVault.Extensions)
+
+## Versioning Information
+
+- The Key Vault SDK uses [the semantic versioning scheme.](http://semver.org/)
+
+## Target Frameworks
+
+For information about the target frameworks of the Key Vault SDK, please refer to the [Target Frameworks](https://github.com/azure/azure-sdk-for-net#target-frameworks) of the Microsoft Azure SDK for .NET.
+
+## Prerequisites
+
+The Key Vault Client Library shares the same [Prerequisites](https://github.com/azure/azure-sdk-for-net#prerequisites) as the Microsoft Azure SDK for .NET.
+
+## To Build
+
+For information on building the Azure Key Vault SDK, please see [Building the Microsoft Azure SDK for .NET](https://github.com/azure/azure-sdk-for-net#to-build)
+
+## Running Tests
+
+Tests for the Azure Key Vault SDK are run in the same manner as the rest of the tests for the Azure SDK for .NET.  For information please see, [to run the tests](https://github.com/azure/azure-sdk-for-net#to-run-the-tests)
```

Comment:
(nitpick) I would rephrase this to "please see how to run tests" (with the 'run tests' being hyperlinked).